nss-day-cohort-41/tabloid-fullstack-bat-boy-apologists
# fixed bug with deleting in use tags

# Description
Fixed bug when user attempted to delete a tag that was assigned to a post. Will now change the tag to be deleted to "Other".
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Add an "Other" tag to your tag list, either in SQL or in the app.
Delete a tag that is in use. Make sure that 1. It doesn't blow up and 2. It changes the label on the post to "Other"
# Checklist:
